examples/3d/edit_material_on_gltf.rs

```diff
@@ -0,0 +1,89 @@
+//! Showcases how to change the material of a `Scene` spawned from a Gltf
+
+use bevy::{
+    app::{App, PluginGroup, Startup},
+    asset::{AssetServer, Assets},
+    audio::AudioPlugin,
+    color::{palettes, Color},
+    gltf::GltfAssetLabel,
+    math::{Dir3, Vec3},
+    pbr::{DirectionalLight, MeshMaterial3d, StandardMaterial},
+    prelude::{Camera3d, Children, Commands, Component, Query, Res, ResMut, Transform, Trigger},
+    scene::{SceneInstanceReady, SceneRoot},
+    DefaultPlugins,
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.build().disable::<AudioPlugin>())
+        .add_systems(Startup, setup_scene)
+        .add_observer(change_material)
+        .run();
+}
+
+/// Overrides the color of the `StandardMaterial` of a mesh
+#[derive(Component)]
+struct ColorOverride(Color);
+
+fn setup_scene(mut commands: Commands, asset_server: Res<AssetServer>) {
+    // Camera
```

Comment:
These comments don't seem super useful. I'd suggest removing the camera and directional light comments and adding e.g.

```
// This scene will keep its original materials
// These scenes will have their materials overridden with the specified color
```